extensions/amp-story/1.0/amp-story.js

```diff
@@ -1489,15 +1489,26 @@ export class AmpStory extends AMP.BaseElement {
    * @param {!Object} size including new width and height
    * @private
    */
-  polyfillDvh_(size) {
+  polyfillDvh_(size, pageId) {
```

Comment:
Please create another method to set the `PAGE_SIZE`: `pageId` and `PAGE_SIZE` don't belong in `polyfillDvh_` as they're different things